content/manuals/scout/policy/_index.md

```diff
@@ -109,31 +117,32 @@ The following policy parameters are configurable in a custom version:
 
 For more information about configuring policies, see [Configure policies](./configure.md).
 
-### No AGPL v3 licenses
+### Compliant Licenses
```

Comment:
**thought (non-blocking):** I've noticed some of the policy types do mention their default configuration but this one doesn't. We should probably include a description here and any other policy that doesn't mention what it does by default. Another proposal is to take those descriptions into a new section that describes the default set of policies that new orgs get. Should keep things separated. Maybe as a next step. Don't want to block this PR.